### ISSUE TITLE
Broken in lessc 3.7

### ISSUE THREAD

**Issue by CliffS** [seven-phases-max/less-plugin-lists#11]
The `.for-each` command works up to and including lessc version 3.6.0.

In 3.7.0 and current it gives:

```
ParseError: Unrecognised input in for-each.less on line 6, column 33:
5 #for-each-fruit {
6     .for-each(@value in @fruit) {
7         value: @value;
```
where the open brace on line 6 is in red.

**Comment by seven-phases-max** [comment]
I'm afraid I'm about to stop support the thing I don't like so any compatibilty issues here are to be counted as Less issues not the plugin issues.

**Comment by seven-phases-max** [comment]
Yes, this is something changed in Less parser in recent versions (I guess it was a fix for https://github.com/less/less.js/issues/2824). Unfortunately, I'm afraid I won't be able to fix this myself any time soon - so any PRs (or just ideas) are welcome. [Related info of interest](https://github.com/less/less.js/issues/2270#issuecomment-189738910).

Also note that the breaking version of Less already has its own `each` function so you can use it as a replacement (though if I'm not mistaken they are not 100% interchangable and there're non overlapping limitations/pros/cons in either of these facilities).
